### DISCUSSION TITLE
Add ability to turn of panel outline when embedding panel using an iFrame

### DISCUSSION THREAD

**Discussion by jeffreymonroe** [grafana/grafana#69969] [Dashboards, Panels & Visualizations]
Embedded Grafana panels have a border which should be able to be removed if embedded in an iFrame with other content on an external site.
![image](https://github.com/grafana/grafana/assets/7051807/8b4f10bc-7899-4fd9-a07b-9c8fed723545)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82653. Feel free to continue the discussion around this there. Thank you!
